pom.xml

```diff
@@ -329,6 +330,11 @@
                 <artifactId>commons-collections4</artifactId>
                 <version>${commons-collections4.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-csv</artifactId>
+                <version>${commons-csv.version}</version>
+            </dependency>
```

Comment:
please add this into https://github.com/apache/seatunnel/blob/e1a81ac1677166e07c21311966705e0029e79b7b/tools/dependencies/known-dependencies.txt